0. Kirish/0.2 Tamoyillari.md

```diff
@@ -0,0 +1,32 @@
+# 0.2 Tamoyillari
+Ushbu qismda biz Me'yor tamoyillarini o'rganamiz. Ushbu tamoyillar me'yorga mos keladigan tizimning barcha komponentlariga taalluqlidir. Xo'sh ushbu komponentlar brokerlar, xizmatlar yoki taqdim etuvchilar bo'ladimi?
```

Comment:
`These principles apply to all components in a Standard-compliant system, whether these components are brokers, services, or exposers.`
Bu gapni siz quyidagicha tarjima qilibsiz. Vaholanki oxirgi gap `xoh ... xoh` ma'nosida kelypati: 
`Ushbu tamoyillar me'yorga mos keladigan tizimning barcha komponentlariga taalluqlidir. Xo'sh ushbu komponentlar brokerlar, xizmatlar yoki taqdim etuvchilar bo'ladimi?`

Quyidagicha o'zgartirishni taklif etaman: 
`Ushbu tamoyillar me'yorga mos keladigan tizimning barcha komponentlariga taalluqlidir, xoh ular brokerlar, xizmatlar yoki taqdim etuvchilar bo'lsin`